app/src/main/java/com/blockeq/stellarwallet/activities/RecoverWalletActivity.kt

```diff
@@ -107,6 +138,41 @@ class RecoverWalletActivity : BaseActivity() {
     //endregion
 
     //region Helper functions
+    private fun highlightMnemonic() {
+        val wordListBIP39 = WordList.ENGLISH.words.toHashSet()
+
+        val tokens = phraseEditText.text.split(" ".toRegex()).dropLastWhile { it.isEmpty() }
+        var startIndex = 0
+        var endIndex = 0 //hi h
+
+        for (word in tokens) {
+
+            // Color the last word
+            endIndex += word.length
+
+            val colorText = if (!wordListBIP39.contains(word)) {
+                ForegroundColorSpan(Color.RED)
+            } else {
+                ForegroundColorSpan(Color.BLACK)
```

Comment:
you are overriding the default color coming from the layout, please try to do not use another hardcoded value here